### ISSUE TITLE
Support changing background

### ISSUE THREAD

**Issue by docwhat** [morhetz/gruvbox#135]
I prefer my background to be `#000000`.  You rejected #102 saying:

> I would try to provide alternative solution to overload any gruvbox color

I would like this feature.

Using the workaround (mentioned in #74) "works" but is fragile; switching `colorscheme` causes it to be lost:

``` .vim
colorscheme gruvbox
highlight Normal guibg=0 ctermbg=0
```

It would be nice to have "official" support for this.

**Comment by kascote** [comment]
@morhetz, yes please... add a feature to tweak the colors, I prefer too a darker background that the one by default.

**Comment by AlessandroYorba** [comment]
@docwhat @kascote 
Here's a solution to tweak a colorscheme without the strange cornercases when switching to another colorscheme. 
```VimL
colo gruvbox
if g:colors_name == "gruvbox"
hi Normal ctermbg=16 guibg=#000000
"List other overrides here
endif

**Comment by CookiesBestDinner** [comment]
I sometimes want to use guibg=NONE, and perhaps it has to do with my $TERM value but the last set background persists when un-setting Normal's guibg. The way I'm dealing with it for the moment is with the following modification in gruvbox:

```viml
" Vanilla colorscheme ---------------------------------------------------------
" General UI: {{{

" Normal text
call s:HL('Normal', s:fg1, s:none)
"                             ^ changed to none
```

And then setting the colorscheme with
```viml
silent! colorscheme gruvbox
```
to suppress a bunch of E420's
